arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,227 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "SqlManagementClient",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete Databases, Servers and related resources.",
+        "version": "2015-05-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
+    ],
+    "produces": [
+        "application/json",
+        "text/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/securityAlertPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_CreateOrUpdateServerSecurityAlertPolicy",
+                "description": "Creates or updates an Azure SQL Server security alert policy.",
+                "parameters": [
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicyCreateOrUpdateParameters"
+                        },
+                        "description": "The required parameters for creating or updating a Azure SQL Server security alert policy."
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "The Server Threat Detection Policy"
+                    },
+                    "400": {
+                        "description": "Bad Request"
+                    },
+                    "404": {
+                        "description": "Resource was not found"
+                    }
```

Comment:
It is recommended to have a default response for every swagger operation. Please look at the documentation over here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#default-response